web-client/src/client/lib.rs

```diff
@@ -163,14 +163,9 @@ impl Client {
 
         // Create client from config.
         log::info!("Initializing light client");
-        let mut client = nimiq::client::Client::from_config(
-            config,
-            Box::new(|fut| {
-                spawn_local(fut);
-            }),
-        )
-        .await
-        .expect("Client initialization failed");
+        let mut client = nimiq::client::Client::from_config(config)
+            .await
+            .expect("Client initialization failed");
         log::info!("Web client initialized");
 
         // Start consensus.
```

Comment:
@styppo the `web-client` crate has a bunch of `wasm_bindgen::spawn_local`s. Those can get replaced with the new `spawn` abstraction too.

Reply:
In that case it is guaranteed for it to not be a `tokio::spawn` though as the web client will be compiled for the wasm target always. It could be done, but it would not add much I guess.

Reply:
That's true but for consistency across the codebase using the abstraction as much makes sense. For example if you want to alter the impl behind the abstraction maybe later in the far future.